docs/BasicCommunication/GetSystemTime/index.md

```diff
@@ -0,0 +1,109 @@
+## GeSystemTime
+
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: Obtain current UTC time  
+
+When an application  wants to open protected service it sends certificate to SDL during TLS or DTLS handshake.  
+In order to validate the certificate and to check whether this certificate is not expired SDL needs system time.
+
+### Request
+SDL sends GetSystemTime request after receiving BC.OnSystemTimeReady from HMI.  
+After sending the request SDL starts [`<DefaultTimeout>`](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/develop/docs/Configuration%20file/index.md#main) (value from .ini file) waiting for response from HMI. 
+
+#### Parameters
+
+### Response
+
+!!! must  
+
+1.	Send valid response during `<DefaultTimeout>`(value from .ini file)
+2.	Provide all parameters of "DateTime" struct at response (“milliseconds”,” tz_hour”,” tz_minute” params should be set to “0” as QNX systemTime struct has NO these params)
+
+_Note:_ SDL logs corresponding error internally and fails TLS Handshake if at least one of the following failures happens:
```

Comment:
Please add DTLS: In case TLS or DTLS handshake fails...

Reply:
Done